content/200-concepts/100-components/01-prisma-schema/07-referential-actions.mdx

```diff
@@ -0,0 +1,315 @@
+---
+title: 'Referential actions'
+metaTitle: 'Referential actions'
+metaDescription: 'Referential actions let you define the update and delete behavior of related models on the database level'
+tocDepth: 2
+preview: true
+---
+
+<TopBlock>
+
+As of **2.26.0** Prisma now supports referential actions as part of the Prisma Schema Language.
+
+This is a [**preview feature**](../../../about/releases#preview) and can be accessed by:
+
+1. [Upgrading to version v2.26.0 of Prisma](../../../guides/upgrade-guides/upgrading-to-latest).
+2. Adding `referentialActions` to the `previewFeatures` field of the [`generator`](../../../reference/api-reference/prisma-schema-reference#generator) block in the `prisma.schema` file.
+
+```prisma file=schema.prisma
+generator client {
+    provider = "prisma-client-js"
+    previewFeatures: ['referentialActions']
+}
+```
+
+You now have 3 options:
+
+1. Make your schema reflect the current state of the database by running an [Introspection](../introspection). You can now opt in and use the new referential actions if you wish.
+2. Define custom referential actions on your data model relation fields. You then update the database schema by running a [Migration](../prisma-migrate) or the [`db push`](../prisma-migrate/db-push) command.
+3. Use the new default referential actions that exist when no custom actions are added manually by running a [Migration](../prisma-migrate).
+
+To begin using referential actions you add them to the `@relation` attribute in your schema.
+
+```prisma
+@relation(fields: [authorId], references: [id], onDelete: Restrict)
```

Comment:
Maybe something other than `Restrict`, due to it being a default and this would get removed when introspecting. Better would be: `Cascade`.